pages/linux/needrestart.md

```diff
@@ -0,0 +1,36 @@
+# needrestart
+
+> Checks which daemons need to be restarted after library upgrades.
+> More information: <https://github.com/liske/needrestart>.
+
+- List outdated processes:
+
+`needrestart`
+
+- Interactively restart services:
+
+`sudo needrestart`
+
+- Run in [v]erbose or [q]uiet mode:
+
+`needrestart -{{v|q}}`
+
+- Check if the kernel is obsolete:
+
+`needrestart -k`
+
+- Check if the CPU microcode is obsolete:
+
+`needrestart -w`
+
+- Display output in batch mode:
+
+`needrestart -b`
+
+- Run using a configuration file:
+
+`needrestart -c {{path/to/config}}`
+
+- Display help:
```

Comment:
Not mandatory, but personally, I like it better that way.
```suggestion
- Display help for `needrestart`:
```

Reply:
Didn't we sort of remove this and always just use `display help` a little while ago?

Reply:
Not sure, but could be. But isn't that an essential part of being tldr maintainer? Making a PR to change something and then making another PR to change it back? 😆